src/git-remove-all.js

```diff
@@ -16,20 +15,41 @@ async function lsFiles(options) {
   return files;
 }
 
+function chunkFilePaths(files, maxChunkSize = 4096) {
```

Comment:
Perhaps we should get a unit test or two on this to prove that it is chunking as expected. I would hate to tweak this in the future and accidentally break it.

Reply:
👍good idea, I'll make one and then update